### PR TITLE
Fix/step by step nav

### PR DESCRIPTION
**Pull request by lks-mtzn**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: src/Services/CategoryService.php [hunk 1]
```diff
@@ -211,21 +215,42 @@ public function getCurrentCategoryChildren($categoryId = null)
...
+
+            if ($isInBranch && $category['id'] == $categoryId) {
+                $result = $category;
+                break;
+            }
+
+            if ($isInBranch && count($category['children'])) {
+                $result = $this->findInCategoryTree($category['children'], $branch, $level + 1);
                 break;
             }
```
Comment (stentrop) [inline src/Services/CategoryService.php]:
```suggestion
            if ($category['id'] !== $branch[$branchKey]) {
                continue;
            } elseif ($category['id'] == $categoryId) {
                $result = $category;
                break;
            } elseif (count($category['children'])) {
                $result = $this->findInCategoryTree($category['children'], $branch, $level + 1);
                break;
            }
```

**Review by stentrop — CHANGES_REQUESTED** [review]

**Review by stentrop — APPROVED** [review]

**Comment by lks-mtzn** [comment]
.restart Tests

**Comment by lks-mtzn** [comment]
Restart Tests.

**Comment by lks-mtzn** [comment]
Restart tests.

**Pull request merged**